src/Microsoft.AspNet.Server.Kestrel/App.config

```diff
@@ -0,0 +1,4 @@
+﻿<configuration>
+  <dllmap dll="libuv.dll" target="libuv.so.1" os="linux"/>
+  <dllmap dll="libuv.dll" target="native/darwin/universal/libuv.dylib" os="osx"/>
```

Comment:
This is a special file for library mapping in mono. See http://www.mono-project.com/docs/advanced/pinvoke/#library-names